Migrate iOS 16 expectations to OpenSource

robert-jenner opened the pull request:
#### 4a63e729b347ed4300dd24b2f159fe47261aad17
<pre>
Migrate iOS 16 expectations to OpenSource
<a href="https://bugs.webkit.org/show_bug.cgi?id=242650">https://bugs.webkit.org/show_bug.cgi?id=242650</a>

Reviewed by NOBODY (OOPS!).

* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Added.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/masking-path-04-b-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/use-invalid-pattern-expected.txt: Added.
* LayoutTests/platform/ios/svg/custom/use-invalid-style-expected.txt: Added.
* LayoutTests/platform/ios/svg/hixie/error/012-expected.txt:
* Tools/Scripts/webkitpy/port/ios.py:
(IOSPort):
* Tools/Scripts/webkitpy/port/ios_device_unittest.py:
* Tools/Scripts/webkitpy/port/ios_simulator_unittest.py:
(IOSSimulatorTest.make_port):
(IOSSimulatorTest.test_layout_test_searchpath_with_apple_additions):
(IOSSimulatorTest.test_layout_test_searchpath_without_apple_additions):
(IOSSimulatorTest.test_layout_searchpath_wih_device_type):
* Tools/Scripts/webkitpy/port/ios_testcase.py:
(IOSTest.make_port):
</pre>

aproskuryakov commented on LayoutTests/platform/ios/TestExpectations:
```diff
@@ -3676,3 +3676,13 @@ webkit.org/b/242273 fast/events/ios/dragstart-on-image-by-long-pressing.html [ P
 http/tests/media/media-source/mediasource-rvfc.html [ Timeout ]
+
+# rdar://93063355 ([ iOS16 ] 8x imported/w3c/web-platform-tests/mathml/relations/* layout tests are a constant text failure)
+imported/w3c/web-platform-tests/mathml/relations/css-styling/display-2.html [ Pass Failure ]
```
As these are a constant failure, why is this expectation flaky? Is your goal to avoid adding an ios-15 expectations directory?

robert-jenner replied:
I have fixed this in my latest edits to this PR. I'm having it go through EWS again just to verify my changes worked.

fujii committed:
Migrate iOS 16 expectations to OpenSource
https://bugs.webkit.org/show_bug.cgi?id=242650

Reviewed by NOBODY (OOPS!).

* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/cookies/value/value-ctl-expected.txt: Added.
* LayoutTests/platform/ios/imported/w3c/web-platform-tests/html/dom/documents/resource-metadata-management/document-cookie-expected.txt: Added.
* LayoutTests/platform/ios/svg/W3C-SVG-1.1/masking-path-04-b-expected.txt:
* LayoutTests/platform/ios/svg/batik/text/textStyles-expected.txt:
* LayoutTests/platform/ios/svg/custom/use-invalid-pattern-expected.txt: Added.
* LayoutTests/platform/ios/svg/custom/use-invalid-style-expected.txt: Added.
* LayoutTests/platform/ios/svg/hixie/error/012-expected.txt:
* Tools/Scripts/webkitpy/port/ios.py:
(IOSPort):
* Tools/Scripts/webkitpy/port/ios_device_unittest.py:
* Tools/Scripts/webkitpy/port/ios_simulator_unittest.py:
(IOSSimulatorTest.make_port):
(IOSSimulatorTest.test_layout_test_searchpath_with_apple_additions):
(IOSSimulatorTest.test_layout_test_searchpath_without_apple_additions):
(IOSSimulatorTest.test_layout_searchpath_wih_device_type):
* Tools/Scripts/webkitpy/port/ios_testcase.py:
(IOSTest.make_port):
